cea/tests/test_config.py

```diff
@@ -55,6 +55,10 @@ def test_applying_parameters(self):
         config = pickle.loads(pickle.dumps(config))
         self.assertEquals(config.scenario, config.general.scenario)
 
+    def test_decode_fileparameter(self):
+        config = cea.config.Configuration()
+        expected_output = "/projects/heating/inputs/building-geometry/zone.shp"
```

Comment:
this should be pointing to a generic path, the beginning should be smth like `config.project/config.general.scenario/inputs` instead of the fixed path in your PC

Reply:
Of course :) thanks @shanshanhsieh